src/pages/RFOX/hooks/useCurrentEpochRewardsQuery.ts

```diff
@@ -33,15 +31,24 @@ export const useCurrentEpochRewardsQuery = ({
     () =>
       stakingAssetAccountAddress
         ? async () => {
+            const epochHistory = await queryClient.fetchQuery({
+              queryKey: getEpochHistoryQueryKey(),
+              queryFn: epochHistoryQueryFn,
+            })
+
+            const currentEpochMetadata = epochHistory[epochHistory.length - 1]
+
+            if (!currentEpochMetadata) throw new Error('No current epoch metadata')
+
```

Comment:
[((probably??)) blocking]: By the looks, `epochHistoryQueryFn` will iterate epochs until the last historical epoch prior to the current one, at least that was the intent. We'll either need to extend the existing `epochHistoryQueryFn` to also fetch the current one (which IMO isn't historical and affects the other hooks - caution required), or create another function/query to create an `EpochMetadata` for the current epoch.

Reply:
> "My Rewards Balance" / "Pending Rewards Balance" correctly load, to zero rewards balance eventually since rewards haven't started yet.

Might explain why it's not easy to detect during development

Reply:
Just remembered, we should be able to leverage `src/pages/RFOX/hooks/useCurrentEpochRewardsQuery.ts` to get the `distributionAmountRuneBaseUnit` and create a `EpochMetadata` for the current epoch.

Reply:
https://github.com/shapeshift/web/pull/7204/commits/f6b662a443e1b1340aa4c95375cdf21a7f5cdcf2 - note the addition of `PartialEpochMetadata`, since we cannot get blocks by timestamp in the future (i.e the end block of the current epoch, which is always going to be in the future), but also don't happen to actually need these at consumption time for current epoch